pkg/images/image_utils.go

```diff
@@ -225,23 +225,69 @@ func GetImage(imageNames map[string]string, imageRegistry string, enableSHA bool
 	return skupperImage
 }
 
-func GetImages(component string, enableSHA bool) []SkupperImage {
+func GetImages(component string, enableSHA bool, runningPods map[string]string) []SkupperImage {
 	//var names map[string]string
 	var registry string
 
 	names := make(map[string]string)
 	switch component {
 	case "router":
 		// skupper router has two components
-		names[RouterImageEnvKey] = RouterImageName
-		names[AdaptorImageEnvKey] = AdaptorImageName
-		registry = GetImageRegistry()
+		runningImage := runningPods["router"]
+		envImage := os.Getenv(RouterImageEnvKey)
+		defaultImage := RouterImageName
```

Comment:
do we need defaultImage variable.   Can we just assign names[RuterImageEnvKey] = RouterImageName

Reply:
solved! (aeb4da27f4b1bbf947326063d45f14e3a6655504)